nessie/sql_templates/index_advising_notes.template.sql

```diff
@@ -66,7 +66,8 @@ SELECT ascn.sid, ascn.id, NULL AS note_body, NULL AS advisor_sid, ascn.advisor_u
 FROM {rds_schema_asc}.advising_notes ascn
 UNION
 SELECT dsn.sid, dsn.id, dsn.body AS note_body, dsna.sid AS advisor_sid, dsna.uid AS advisor_uid, dsna.first_name AS advisor_first_name,
-       dsna.last_name AS advisor_last_name, NULL AS note_category, NULL AS note_subcategory, NULL AS created_by, dsn.created_at, NULL AS updated_at
+       dsna.last_name AS advisor_last_name, NULL AS note_category, NULL AS note_subcategory, NULL AS created_by, dsn.created_at,
+       dsn.created_at AS updated_at
```

Comment:
@lyttam - `created_at AS updated_at` is intended?

Reply:
@johncrossman yes, the original data doesn't have an `updated_at` field but it's needed for searching these notes in BOA.